src/rpcclient.cpp

```diff
@@ -194,10 +194,31 @@ static const CRPCConvertParam vRPCConvertParams[] =
     { "getblockbynumber"       , 0 },
     { "getblockbynumber"       , 1 },
     { "getblockhash"           , 0 },
-    { "listpollresults"        , 1 },
+    { "listpolls"              , 0 },
     { "setban"                 , 2 },
     { "setban"                 , 3 },
     { "showblock"              , 0 },
+    { "votebytxid"             , 1 },
+    { "votebytxid"             , 2 },
+    { "votebytxid"             , 3 },
+    { "votebytxid"             , 4 },
+    { "votebytxid"             , 5 },
+    { "votebytxid"             , 6 },
+    { "votebytxid"             , 7 },
+    { "votebytxid"             , 8 },
+    { "votebytxid"             , 9 },
+    { "votebytxid"             , 10 },
+    { "votebytxid"             , 11 },
+    { "votebytxid"             , 12 },
+    { "votebytxid"             , 13 },
+    { "votebytxid"             , 14 },
+    { "votebytxid"             , 15 },
+    { "votebytxid"             , 16 },
+    { "votebytxid"             , 17 },
+    { "votebytxid"             , 18 },
+    { "votebytxid"             , 19 },
+    { "votebytxid"             , 20 },
+    { "votebytxid"             , 21 },
```

Comment:
Hehehe.

Reply:
Could not think of a better way to set that up...